docs/user_guides/fs/feature_group/create.md

```diff
@@ -68,7 +68,7 @@ Additionally we specify which columns of the DataFrame will be used as primary k
 
 The version number is optional, if you don't specify the version number the APIs will create a new version by default with a version number equals to the highest existing version number plus one. 
 
-The last parameter used in the examples above is `stream`. The `stream` parameter controls whether to enable the streaming write APIs to the online and offline feature store. When using the APIs in a Python environment this behavior is the default. 
+The last parameter used in the examples above is `stream`. The `stream` parameter controls whether to enable the streaming write APIs to the online and offline feature store. When using the APIs in a Python environment this behavior is the default. Furthermore, for the stream feature group, the time travel format is always HUDI.  
```

Comment:
`stream feature group` is an internal representation, it's not exposed to the user and we don't use it in the documentation